MAUI/Popup/popup-positioning.md

```diff
@@ -268,6 +274,80 @@ public class ViewModel : INotifyPropertyChanged
 
 ## How to
 
+### Returning result
+
+The `ShowAsync()` method of the Popup allows you to display the popup and returns `true` if the user closes it using the accept button, or `false` otherwise.
+
+{% tabs %}
+{% highlight xaml %}
+<?xml version="1.0" encoding="utf-8" ?>
+<ContentPage xmlns="http://schemas.microsoft.com/dotnet/2021/maui"
+             xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+             xmlns:local="clr-namespace:PopupDemo"
+             xmlns:sfPopup="clr-namespace:Syncfusion.Maui.Popup;assembly=Syncfusion.Maui.Popup"
+             x:Class="PopupDemo.MainPage">
+    <StackLayout padding="20">
+        <Button x:Name="clickToShowPopup"
+                Text="ClickToShowPopup"
+                VerticalOptions="Start"
+                HorizontalOptions="Center"
+                Clicked="ClickToShowPopup_Clicked" />
+        <sfPopup:SfPopup x:Name="popup">
+        </sfPopup:SfPopup>
+    </StackLayout>
+</ContentPage>
+{% endhighlight %}
+{% highlight c# tabtitle="MainPage.xaml.cs" hl_lines="12" %}
+using Syncfusion.Maui.Popup;
+public partial class MainPage : ContentPage
+{
+  SfPopup popup;
+  public MainPage()
+  {
+    InitializeComponent();
+    popup = new SfPopup();
+  }
+  private async void ClickToShowPopup_Clicked(object sender, EventArgs e)
+  {
+    bool answer = await popup.ShowAsync();
+  }
+}
+{% endhighlight %}
+{% endtabs %}
+
+The `Show(string title, string message, string acceptText, string declineText)` static method of the Popup allows you to display the popup and returns `true` if the user closes it using the accept button, or `false` otherwise.
+
+{% tabs %}
+{% highlight xaml tabtitle="MainPage.xaml" %}
+<ContentPage xmlns="http://schemas.microsoft.com/dotnet/2021/maui"
+             xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+             xmlns:sfPopup="clr-namespace:Syncfusion.Maui.Popup;assembly=Syncfusion.Maui.Popup"
+             x:Class="PopupMauiPositioning.MainPage">
+  <ContentPage.Content>
+    <StackLayout Padding="20">
+      <Button x:Name="clickToShowPopup" Text="ClickToShowPopup" 
+              VerticalOptions="Start" HorizontalOptions="Center" 
+              Clicked="ClickToShowPopup_Clicked" />
+    </StackLayout>
+  </ContentPage.Content>
+</ContentPage>
+{% endhighlight %}
+{% highlight c# tabtitle="MainPage.xaml.cs" hl_lines="10" %}
+using Syncfusion.Maui.Popup;
+public partial class MainPage : ContentPage
+{
+  public MainPage()
+  {
+    InitializeComponent();
+  }
+  private async void ClickToShowPopup_Clicked(object sender, EventArgs e)
+  {
+    bool answer = await SfPopup.Show("Question?", "Would you like to play a game", "Yes", "No");
+  }
+}
+{% endhighlight %}
+{% endtabs %}
+
 ### Load the SfPopup in CellTappedEvent of the SfDataGrid
 
 The `SfPopup` allows opening it in the [CellTapped](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.DataGrid.SfDataGrid.html#Syncfusion_Maui_DataGrid_SfDataGrid_CellTapped) event of the `SfDataGrid`.

```

Comment:
The `SfPopup` allows opening it in the [CellTapped](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.DataGrid.SfDataGrid.html#Syncfusion_Maui_DataGrid_SfDataGrid_CellTapped) event of the `SfDataGrid.`